Makefile

```diff
@@ -17,7 +17,7 @@ test-integ: test
 
 version ?= latest
 test-docker:
-	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/HEAD/prism/prism.sh -o prism.sh
+	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/sendgrid-java-cert/prism/prism.sh -o prism.sh
```

Comment:
Points sendgrid-java to the `sendgrid-java-cert` branch of sendgrid-oai, which contains a [non-SAN cert](https://github.com/sendgrid/sendgrid-oai/tree/sendgrid-java-cert/prism/nginx) that plays nice with the unit tests

Reply:
Since we're having to modify both this file and the actual `prism.sh` file, I'd prefer to not use a branch and instead create a `prism-java.sh` file on the main branch which clones the oai repo and swaps out the non-java cert with the java cert.

Reply:
That sounds like a better plan